scripts/build-linux.sh

```diff
@@ -4,4 +4,6 @@ echo "BUILDING WITH VERSION: ${VERSION}"
 
 wails build -platform linux/amd64 -clean -ldflags "-s -w -X github.com/SafetyCulture/safetyculture-exporter-ui/internal/version.version=${VERSION}"
 
-tar -czf exporter-linux-amd64.tar.gz ./build/bin/safetyculture-exporter
+mv ./build/bin/safetyculture-exporter .
+
+mv safetyculture-exporter exporter-linux-amd64
```

Comment:
what was the problem here?

Reply:
No problem. We are not zipping the executables.

Reply:
why not zipping the executables? Or is less friction for the user?

Reply:
Yes. less friction for the user.

Reply:
Had a chat with Austin and not zip is fine